spec/support/vcr_cassettes/claims_api/v1/disability_comp/invalid.yml

```diff
@@ -0,0 +1,83 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: "https://fakedomainandsubdomain.com/wss-form526-services-web/rest/form526/v2/validate"
+    body:
+      encoding: UTF-8
+      string: ''
+    headers:
+      User-Agent:
+      - Faraday v0.9.2
+      va-eauth-csid:
+      - DSLogon
+      va-eauth-authenticationmethod:
+      - DSLogon
+      va-eauth-pnidtype:
+      - SSN
+      va-eauth-assurancelevel:
+      - '3'
+      va-eauth-firstName:
+      - WESLEY
+      va-eauth-lastName:
+      - FORD
+      va-eauth-issueinstant:
+      - '2017-12-07T00:55:09Z'
+      va-eauth-dodedipnid:
+      - '4789224336'
+      va-eauth-birlsfilenumber:
+      - '796068948'
+      va-eauth-pid:
+      - '7420876015'
+      va-eauth-pnid:
+      - '796068949'
+      va-eauth-birthdate:
+      - '1959-10-24T00:00:00+00:00'
+      va-eauth-authorization:
+      - '{"authorizationResponse":{"status":"VETERAN","idType":"SSN","id":"796068949","edi":"4789224336","firstName":"Beyonce","lastName":"Knowles","birthDate":"1959-10-24T00:00:00+00:00","gender":"FEMALE"}}'
+      Content-Type:
+      - application/json
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+  response:
+    status:
+      code: 400
+      message:
+    headers:
+      Date:
+      - Mon, 02 Apr 2018 22:46:34 GMT
+      Server:
+      - Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips
+      Content-Type:
+      - application/json
+      Set-Cookie:
+      - WLS_12.1_App1_Cluster_ROUTEID=.02; path=/
+      - WSS-FORM526-SERVICES_JSESSIONID=RPeIimX_DLhskhO20bPzBmWWKGghuW-YlSjLgIX3nN28Adki1xRE!-1392655551;
+        path=/; HttpOnly
+      Via:
+      - 1.1 csraciapp6.evss.srarad.com
+      Transfer-Encoding:
+      - chunked
+    body:
+      encoding: UTF-8
+      string: |-
+        { "messages": 
+          [
+            {
+              "key": "form526.serviceInformation.servicePeriods[1].serviceBranch.isInvalidValue",
+              "severity": "ERROR",
+              "text": "The branch of service for the claimant's period of service is not a valid value"
+            },
+            {
+              "key": "form526.directDeposit.isNull",
+              "severity": "WARN",
+              "text": "Direct deposit information was not provided"
+            }
+          ]
+        }
+    http_version:
+  recorded_at: Mon, 02 Apr 2018 22:46:34 GMT
+recorded_with: VCR 3.0.3
+
```

Comment:
do we need an entirely new cassette here? technically we could just modify the `request.uri`.  

I'm curious as well, the same call in the previous test recorded a 200 with the error messages, but we're seeing a 400 here. Is that a discrepancy between the EVSS code and the EVSS Docker Container? seems like we should be returning the same `response.status.code` ultimately.

Reply:
@FonzMP There is not a discrepancy. I created this cassette manually to update the URL since I was unsure if changing the original would have an rippling side effects since it is used in another file. I suspect I just messed up by setting it as 400 instead of 200. We do not need both here so I will clean that up.